astropy/convolution/convolve.py

```diff
@@ -672,7 +665,7 @@ def convolve_fft(array, kernel, boundary='fill', fill_value=0.,
             # add the shape lists (max of a list of length 4) (smaller)
             # also makes the shapes square
             fsize = 2 ** np.ceil(np.log2(np.max(arrayshape + kernshape)))
-        newshape = np.array([fsize for ii in range(array.ndim)], dtype=int)
+        newshape = np.array([fsize for _ in range(array.ndim)], dtype=int)
```

Comment:
Is this a widely-used convention?  I don't have a problem with it, just curious

Reply:
Usually when there is a loop like this, the loop itself is unnecessary in most cases. Why not do something like this instead?
```python
newshape = np.full((array.ndim, ), fsize, dtype=int)
```

Reply:
yeah, my ide was complaining with ``ii`` not being used, but I didn't spend any brain power to something better like @pllim suggests. I've seen `_` being used for unnecessary variables several places, either in loops or in unpackings, but I'm not sure how much "official" it is.